webui/src/lib/components/repository/commits.jsx

```diff
@@ -126,12 +126,16 @@ const CommitInfo = ({ repo, commit }) => {
   );
 };
 
+export const CommitMessage = ({ commit }) =>
+    commit.message && commit.message.length ?
```

Comment:
I think JavaScript (and even TypeScript!) would let you say
```suggestion
    commit.?message.?length ?
```